acl_loader/main.py

```diff
@@ -402,16 +403,18 @@ def validate_actions(self, table_name, action_props):
             # Same information should be there in all state DB's
             # as it is static information about switch capability
             namespace_statedb = list(self.per_npu_statedb.values())[0]
-            capability = namespace_statedb.get_all(self.statedb.STATE_DB, "{}|switch".format(self.SWITCH_CAPABILITY_TABLE))
+            aclcapability = namespace_statedb.get_all(self.statedb.STATE_DB, "{}|{}".format(self.ACL_STAGE_CAPABILITY_TABLE, stage.upper()))
```

Comment:
Where is the entry ```ACL_STAGE_CAPABILITY_TABLE``` written into state_db? I didn't find the code in aclorch.

Reply:
It is here https://github.com/Azure/sonic-swss/blob/743027b840d4c82917d735937d07114b5b66f302/orchagent/aclorch.cpp#L2716.
From PR - https://github.com/Azure/sonic-swss/pull/1982

Reply:
I see. Thanks.